R/to-vega.R

```diff
@@ -67,5 +67,9 @@ vw_to_vega <- function(spec) {
    spec
 }
 
-
-
+# Alternate way of doing this via V8
```

Comment:
I have taken this bit as inspiration (maybe more than inspiration) in my branch - so far so good, and it gets us into V8!